libpod/oci.go

```diff
@@ -789,9 +789,10 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 	logrus.Debugf("Starting runtime %s with following arguments: %v", r.path, args)
 
 	execCmd := exec.Command(r.path, args...)
-	execCmd.Stdout = os.Stdout
-	execCmd.Stderr = os.Stderr
-	execCmd.Stdin = os.Stdin
+	execCmd.Stdout = streams.OutputStream
```

Comment:
For good form, we should only set this if streams.AttachOutput is true (same for the others). Maybe default them to /dev/null?

Reply:
If the reader/writer is nil it does nothing (effectively sending to /dev/null). I think I'd prefer have go library take care of it rather than have to open the file manually. WDYT

Reply:
(as in set the stream to nil if Attach* is false)

Reply:
https://golang.org/pkg/os/exec/#Cmd it's actually exactly the same so I'm gonna do that

Reply:
Yeah, nil if fine by me